tests/test_polaraveraging.py

```diff
@@ -9,25 +9,29 @@
 
 
 rtol = 1.e-6
+atol = 1.e-6
 
 
 def test_compute_cross_shear():
-    shear1, shear2, phi = 0.15, 0.08, 0.52
-    expected_cross_shear = -0.08886301350787848
-    cross_shear = pa._compute_cross_shear(shear1, shear2, phi)
-    testing.assert_allclose(cross_shear, expected_cross_shear)
-
-    shear1 = np.array([0.15, 0.40])
-    shear2 = np.array([0.08, 0.30])
-    phi = np.array([0.52, 1.23])
-    expected_cross_shear = [-0.08886301350787848, -0.48498333705834484]
-    cross_shear = pa._compute_cross_shear(shear1, shear2, phi)
-    testing.assert_allclose(cross_shear, expected_cross_shear)
+#    shear1, shear2, phi = 0.15, 0.08, 0.52
```

Comment:
We can remove these instead of just commenting them out now that we have new test cases implemented.